res/database/Default/user/jorgaeff.xml

```diff
@@ -97,42 +111,50 @@
 			<availability script="TIME_YEAR=1985" />
 			<title>
 				<de>APD nominiert Hannes Reif</de>
+				<en>APD nominates Hannes Reif</en>
 			</title>
 			<description>
 				<de>Die Arbeiterpartei Deutschland (APD) nominiert den nordrhein-westfälischen Ministerpräsidenten Hannes Reif zum Kanzlerkandidaten für die Bundestagswahl 1987.</de>
+				<en>The Workers' Party of Germany (APD) nominates Hannes Reif, Minister-President of North Rhine-Westphalia, as its candidate for chancellor in the 1987 federal election.</en>
```

Comment:
Workers' Party not translated in previous entry. Handle consistently?

Reply:
ok